docs/src/components/document-renderer.tsx

```diff
@@ -8,13 +8,16 @@ import { InferRenderersForComponentBlocks } from '@keystatic/core';
 import { componentBlocks } from '../../keystatic.config';
 import { CONTENT_MAX_WIDTH_DESKTOP } from '../constants';
 import CloudImage from './cloud-image';
+import fs from 'fs';
 
 export default async function DocumentRenderer({
   slug,
   document,
 }: DocumentRendererProps & { slug: string }) {
   const highlighter = await shiki.getHighlighter({
-    theme: 'min-light',
+    theme: JSON.parse(
+      fs.readFileSync('./src/styles/keystatic-theme.json', 'utf-8')
+    ),
   });
```

Comment:
Very sorry as this is my dodgy code, but can you update this to move the reading of the file outside the function? Will give a perf boost. 

Github won't let me add this as a suggestion for some reason:

```
const codeTheme = JSON.parse(
  fs.readFileSync('./src/styles/keystatic-theme.json', 'utf-8')
);

export default async function DocumentRenderer({
  slug,
  document,
}: DocumentRendererProps & { slug: string }) {
  const highlighter = await shiki.getHighlighter({
    theme: codeTheme,
  });
```

Reply:
Ahh yes of course!!